Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/ConfigParser.kt

```diff
@@ -14,17 +16,24 @@ class ConfigParser @Inject constructor(
     private val cwaConfigMapper: CWAConfig.Mapper,
     private val keyDownloadConfigMapper: KeyDownloadConfig.Mapper,
     private val exposureDetectionConfigMapper: ExposureDetectionConfig.Mapper,
-    private val riskCalculationConfigMapper: RiskCalculationConfig.Mapper
+    private val riskCalculationConfigMapper: RiskCalculationConfig.Mapper,
+    private val exposureWindowRiskLevelConfigMapper: ExposureWindowRiskLevelConfig.Mapper
 ) {
 
     fun parse(configBytes: ByteArray): ConfigMapping = try {
+        // TODO replace with actual v2 config
+        val dummyConfig = AppConfigAndroid
+            .ApplicationConfigurationAndroid
+            .newBuilder()
+            .build()
         parseRawArray(configBytes).let {
             DefaultConfigMapping(
                 rawConfig = it,
                 cwaConfig = cwaConfigMapper.map(it),
                 keyDownloadConfig = keyDownloadConfigMapper.map(it),
                 exposureDetectionConfig = exposureDetectionConfigMapper.map(it),
-                riskCalculationConfig = riskCalculationConfigMapper.map(it)
+                riskCalculationConfig = riskCalculationConfigMapper.map(it),
+                exposureWindowRiskLevelConfig = exposureWindowRiskLevelConfigMapper.map(dummyConfig)
```

Comment:
adjust variable name if you adjust the class name (exposureWindowRiskCalculationConfig)

Reply:
done 👍